cli/commands/project.go

```diff
@@ -50,7 +50,7 @@ func createProjectHandler(ctx *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("could not initialize client: %s", err)
 	}
-	p := ctx.String(name(pathFlag))
+	p := ctx.String(name(path))
```

Comment:
i meant the receivinbg variable name `p` --> `path`

Reply:
Gotcha!